Pocket.xcodeproj/project.pbxproj

```diff
@@ -561,6 +563,7 @@
 			name = "Tests iOS";
 			packageProductDependencies = (
 				167F8083264B011600D8F507 /* Sails */,
+				8A3B77BC29E0AEEB00AD71D0 /* PocketKit */,
```

Comment:
this change is due to issue mentioned here:
https://pocket.slack.com/archives/C01UH57EJKD/p1680891486803859